### PR TITLE
Adding License file to the project

### PR DESCRIPTION
**Pull request by kshithijiyer**

### CODE REVIEW

**Comment by brentzundel** [comment]
In order for us to merge this PR, we need you to meet the DCO requirements. [This link](https://github.com/hyperledger/ursa-rfcs/pull/21/checks?check_run_id=480232462) should give you more information.

**Comment by kshithijiyer** [comment]
> In order for us to merge this PR, we need you to meet the DCO requirements. [This link](https://github.com/hyperledger/ursa-rfcs/pull/21/checks?check_run_id=480232462) should give you more information.

Done

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (brentzundel) [inline LICENSE]:
```suggestion
   Copyright 2018 Hyperledger
```

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2018 Hyperledger
```
Comment (danintel) [inline LICENSE]: Change 'Copyright 2018 Hyperledger' to `Copyright [yyyy] [name of copyright owner]`
This is a template, not a copyright of the license (which is not created by Hyperledger  or The Linux Foundation anyway).

Comment (kshithijiyer) [reply LICENSE]: The previous comment by @brentzundel told me to do otherwise.   What should I do lastly?

Comment (brentzundel) [reply LICENSE]: Apologies, I made the suggestion following the examples [here](https://github.com/hyperledger/ursa/blob/master/LICENSE) and [here](https://github.com/hyperledger/aries/blob/master/LICENSE), but a closer reading of the text indicates my examples are mistaken. Thank you for pointing out the error @danintel .

Comment (brentzundel) [reply LICENSE]: so @kshithijiyer, (I'm sorry) please revert the suggestion I made and we can merge this.

Comment (kshithijiyer) [reply LICENSE]: @brentzundel @danintel Thank for clearing the confusion. Reverted the changes.

**Pull request merged**